src/TimeSelect.jsx

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+var React = require('react');
+var Input = require('react-bootstrap').Input;
```

Comment:
should this be `UIToolkit.Input` instead? So it automatically adds on our tracking and things?